reservation_units/models.py

```diff
@@ -568,6 +568,16 @@ class ReservationUnit(models.Model):
         help_text="Product used for payments",
     )
 
+    payment_accounting = models.ForeignKey(
```

Comment:
Is it overriding information to have this in the `ReservationUnit` and if not exist then get it from the `Unit`? (`ReservationUnit` has fk reference to `Unit`)

Reply:
Yes, this is going to work similarly to merchant. We'll use ReservationUnit's accounting info if it is available, otherwise we fallback to Unit's info.